services/director/src/simcore_service_director/producer.py

```diff
@@ -914,6 +915,7 @@ async def get_service_details(app: web.Application, node_uuid: str) -> Dict:
             ) from err
 
 
+@run_serialized_in_context(target_args=["node_uuid"])
 async def stop_service(app: web.Application, node_uuid: str) -> None:
```

Comment:
I see, so here you will not call to stop a given node if there is already a stop call running ...

Reply:
What happens exactly is that the call to the function will get delayed until the previous one finishes.